min_elevation: 0.13797
max_elevation: 0.31797
movement_threshold: 0.005
command_timeout: 15.0
linear_command_topic: /linear_elevator_controller/command
linear_motor_frame: linear_elevator_link
